src/utils/api.ts

```diff
@@ -0,0 +1,24 @@
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+import { Comment } from '../types/Comment';
+import { client } from './fetchClient';
+
+export const getUsers = () => {
+  return client.get<User[]>('/users');
+};
+
+export const getPosts = (userId: number) => {
+  return client.get<Post[]>(`/posts?userId=${userId}`);
+};
+
+export const getComment = (postId: null | number) => {
```

Comment:
```suggestion
export const getComment = (postId: number) => {
```

Don't make a call where id can be null. If your comment has id equals to null, just don't make a call